tests/phpunit/tests/REST_API/Lock_Controller.php

```diff
@@ -23,9 +23,11 @@
 use WP_REST_Request;
 
 /**
+ * @coversDefaultClass \Google\Web_Stories\REST_API\Lock_Controller
  * @coversDefaultClass \Google\Web_Stories\REST_API\Stories_Lock_Controller
+ * @coversDefaultClass \Google\Web_Stories\REST_API\Template_Lock_Controller
  */
-class Stories_Lock_Controller extends \WP_Test_REST_TestCase {
+class Lock_Controller extends \WP_Test_REST_TestCase {
```

Comment:
I think there should only be one `@coversDefaultClass` annotation here.

PHPUnit seems to complain.

Reply:
Fixed https://github.com/google/web-stories-wp/pull/7225/commits/adcd74e078c2067aa8cea0aa95ca9acccef290b8